### ISSUE TITLE
Setting KEY_OPERATING_RATE and KEY_PRIORITY seems to lower decoding latency

### ISSUE THREAD

**Issue by GregGodin** [polygraphene/ALVRClient#28]
Adding those two options on MediaFormat seems to decrease decoding latency (from 15/20ms to 10ms with default settings) :
        format.setInteger(MediaFormat.KEY_OPERATING_RATE, Short.MAX_VALUE);
        format.setInteger(MediaFormat.KEY_PRIORITY, 0);
Didn't notice side effects, but it would be better with more test from more people.

**Comment by polygraphene** [comment]
Thank you! This is great information.
I could get better latency on native (100%) resolution.
I will check effect on other configurations.

**Comment by sen7inel** [comment]
I tried GregGodin's version and unfortunately at 150% resolution / H264 / 30Mbps I could not discern a difference in latency.  My first .apk compile attempt so it may very well be a problem on my end.

**Comment by MadSpin** [comment]
Sup guys!

I tried GregGodin's version and I was able to get around 10-15ms average drop in latency! I was getting on average before 80ms and now its around 65-75ms. 

I've been running at 3072x1536 res on the Quest then 100% / H265 / 50Mbps / 200k 
Another thing I've been doing is running RTSS (Riva Tuner) to cap the framerate at 72fps. My thought was to lessen the load on the GPU so the encoder can do its thing. It ended up with smoother framerate/gameplay in any scenes in a game that was graphically intensive. Before i would get choppy framerate but now a lot of areas are butter smooth.

Ryzen 7 2700X / 64GB / 2080 TI
